django/core/management/commands/makemigrations.py

```diff
@@ -136,6 +137,13 @@ def handle(self, *app_labels, **options):
             questioner,
         )
 
+        # Check if migration name is a valid
+        if self.migration_name:
+            MIGRATION_NAME_REGEXP = re.compile(r'[^a-zA-Z0-9_]')
+            if MIGRATION_NAME_REGEXP.search(self.migration_name):
```

Comment:
Module names must be identifiers based on Python's documentation

https://docs.python.org/3/reference/simple_stmts.html#grammar-token-module
https://docs.python.org/3/reference/lexical_analysis.html#identifiers
https://docs.python.org/3/library/stdtypes.html#str.isidentifier